rosbag2_test_common/include/rosbag2_test_common/publication_manager.hpp

```diff
@@ -76,26 +79,78 @@ class PublicationManager
 
   void run_publishers(bool verbose = false) const
   {
-    for (const auto & pub_fcn : publishers_) {
+    for (const auto & pub_fcn : publishers_fcn_) {
       pub_fcn(verbose);
     }
+    rclcpp::executors::SingleThreadedExecutor exec;
+    exec.spin_node_some(pub_node_);
+    std::this_thread::sleep_for(100ms);
```

Comment:
Is 100ms meaningful? Where does this number come from? Why do we need to sleep? Does this make the publishing "bursty"?

Reply:
Not necessary. I forgot to remove them.